atomics/include/desul/atomics/Lock_Array_Cuda.hpp

```diff
@@ -10,6 +10,7 @@ SPDX-License-Identifier: (BSD-3-Clause)
 #define DESUL_ATOMICS_LOCK_ARRAY_CUDA_HPP_
 
 #include "desul/atomics/Macros.hpp"
+#include "desul/atomics/Common.hpp"
```

Comment:
What do we need that include for?

Reply:
MemoryScope used further down. This allows me to only include the LockArrayCUDA.hpp standalone. Without all the full atomics. I need that for the Kokkos lock array integration.